src/libraries/MinimalDelegationStorage.sol

```diff
@@ -7,7 +7,7 @@ import {EnumerableSetLib} from "solady/utils/EnumerableSetLib.sol";
 struct MinimalDelegationStorage {
     EnumerableSetLib.Bytes32Set keyHashes;
     mapping(bytes32 keyHash => bytes encodedKey) keyStorage;
-    mapping(uint192 key => uint256 nonce) nonceSequenceNumber;
+    mapping(uint256 key => uint256 seq) nonceSequenceNumber;
```

Comment:
nit but could we just call this nonces ?

the name including "number" is slightly misleading bc the data structure is a mapping

Reply:
hm but i like how its the same here https://github.com/eth-infinitism/account-abstraction/blob/develop/contracts/core/NonceManager.sol#L11-L14

Reply:
i still think i don't like that its "number" when you are going to use that variable locally its misleading what the data type you are working with is.

we don't need to rename in this PR since I wanna just get this in, but I still think I prefer a diff name